teekize/Questionner
# #163061106 Adds the travis.yml file

## What does this PR ?
Adds the travis.yml file for the continous integration

## Description of task to be completed
create a travis.yml file
create a requirement file

## How to manully test this 
1. git clone this repository
(https://github.com/teekize/Questionner)
2.cd into the project folder 
`cd Questionner`
3. git checkout this branch
`git checkout ch-travis-163061106`
4. you will see the travis.yml file

## Screenshot
![traviis](https://user-images.githubusercontent.com/28742266/50885924-1c5e2c00-1400-11e9-8b4a-d84bb590568d.JPG)
